src/utils/string.js

```diff
@@ -42,13 +43,24 @@ export function endsWith (text, search) {
  *     math.format('hello')            // '"hello"'
  *
  * @param {*} value             Value to be stringified
- * @param {Object | number | Function} [options]  Formatting options. See
- *                                                lib/utils/number:format for a
- *                                                description of the available
- *                                                options.
+ * @param {Object | number | Function} [options]
+ *     Formatting options. See src/utils/number.js:format for a
+ *     description of the available options controlling number output.
+ *     This generic "format" also supports the option property `truncate: NN`
```

Comment:
@gwhitney can you also describe the new option `truncate` in the comments of https://github.com/josdejong/mathjs/blob/develop/src/function/string/format.js ? That is the "exposed" function.

I have to say it's a bit messy where we document this, there is also a lot of docs in `/src/util/number.js` for the `format` function that deals with numbers. Mabye we should remove it there. But that's for a separate issue.

Reply:
Sorry I missed that was also an exported function, not just an internal utility. Thanks for the reminder. Should be good now.

Reply:
Thanks!